src/react-apps/applications/runtime/src/shared/resources/applicationMetadata/index.d.ts

```diff
@@ -11,7 +11,7 @@ export interface IApplicationMetadata {
   validTo: string;
   WorkflowId: string;
   maxSize: string;
-  dataTypes: string;
+  dataTypes: any[];
```

Comment:
No typings? Since it went from `string` to `any[]`

Reply:
fixed.

Reply:
Fixed.